alerting/build.gradle

```diff
@@ -198,6 +198,31 @@ integTest {
     }
 }
 
+task integTestRemote(type: RestIntegTestTask) {
```

Comment:
Did you try running this task against a test cluster? 
It would be great to put in a sample run.

Reply:
hi @saratvemulapalli , i have added an example now.